### PR TITLE
Update Set-SPOSite.md

### PR DESCRIPTION
**Pull request by SanjoyanM**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Set-SPOSite [-Identity] <SpoSitePipeBind> [-AllowSelfServiceUpgrade <Boolean>] [
  [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>] [-StorageQuotaReset]
  [-DefaultSharingLinkType] [-DefaultLinkPermission] [-DefaultLinkToExistingAccess]
  [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>] [-LimitedAccessFileType <SPOLimitedAccessFileType>] [-AllowEditing <Boolean>] [-AnonymousLinkExpirationInDays <Int32>] [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>] [-SensitivityLabel <String>][<CommonParameters>]
+ [-RemoveLabel]
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:39]:
```suggestion
 [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>] [-StorageQuotaReset]
 [-DefaultSharingLinkType] [-DefaultLinkPermission] [-DefaultLinkToExistingAccess]
 [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>] [-LimitedAccessFileType <SPOLimitedAccessFileType>] [-AllowEditing <Boolean>] [-AnonymousLinkExpirationInDays <Int32>] [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>] [-SensitivityLabel <String>]  [-RemoveLabel] [<CommonParameters>]
```

**Review by dariomws — APPROVED** [review]
Just a tiny suggestion.

**Review by get-itips — CHANGES_REQUESTED** [review]
This is the PlatyPs output for the parameter, can you add the missing info? (Type, Parameter set, Alias, Accepts...) Thanks!!

### -RemoveLabel
{{ Fill RemoveLabel Description }}

```yaml
Type: SwitchParameter
Parameter Sets: ParamSet5
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

**Review by SanjoyanM — COMMENTED** [review]
I have accepted the changes. Thanks.

**Comment by SanjoyanM** [comment]
@get-itips : I have added description and the bottom of the page

https://github.com/SanjoyanM/office-docs-powershell/blob/patch-7/sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md#-removelabel

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 1 above)
Comment (SanjoyanM) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md]: @dariomws I have made the suggested changes. Thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -893,6 +894,16 @@ Position: Named
 Default value: None
 ```
 
+### -RemoveLabel
+This parameter allows you to remove the assigned sensitivity label on a site.
+
+```yaml
+Type: None
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:901]:
```suggestion
Type: SwitchParameter
Parameter Sets: ParamSet5
Aliases:
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 3]
```diff
@@ -893,6 +894,16 @@ Position: Named
...
+Required: False
+Position: Named
+Default value: None
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:904]:
```suggestion
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 4]
```diff
@@ -36,6 +36,7 @@ Set-SPOSite [-Identity] <SpoSitePipeBind> [-AllowSelfServiceUpgrade <Boolean>] [
  [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>] [-StorageQuotaReset]
  [-DefaultSharingLinkType] [-DefaultLinkPermission] [-DefaultLinkToExistingAccess]
  [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>] [-LimitedAccessFileType <SPOLimitedAccessFileType>] [-AllowEditing <Boolean>] [-AnonymousLinkExpirationInDays <Int32>] [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>] [-SensitivityLabel <String>][<CommonParameters>]
+ [-RemoveLabel] [<CommonParameters>]
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:39]: According to PlatyPs, this parameter belongs to a new Parameter Set named ParamSet5, please see and update, thanks

### ParamSet5
```
Set-SPOSite [-Identity] <SpoSitePipeBind> [-RemoveLabel] [-WhatIf] [-Confirm] [<CommonParameters>]
```

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**